src/components/Collcetion.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-undef */
+import React, {useState, useEffect} from "react";
+import {updateDoc, doc, getDoc, arrayRemove} from "firebase/firestore";
+
+import {
+  AllPinsWrapper,
+  PinWrapper,
+  PinImage,
+  SaveButton as RemoveButton,
+} from "../styles/Homepage.module";
+
+function Collection(props) {
+  const [pinsInCollection, setPinsInCollection] = useState([]);
```

Comment:
prefer having a comment for 'pinsInCollection' data structure. 
like: 
[
    {
         a: 'data',
         b: 'fake data',
     },
     ...
]